tests/test_linear4bit.py

```diff
@@ -186,12 +180,6 @@ def test_linear_serialization(device, quant_type, compress_statistics, bias, qua
 @pytest.mark.parametrize("blocksize", [64, 128])
 @pytest.mark.parametrize("compress_statistics", TRUE_FALSE, ids=id_formatter("compress_statistics"))
 def test_copy_param(device, quant_type, blocksize, compress_statistics):
-    if device == "cpu":
-        if compress_statistics:
-            pytest.skip("Currently segfaults on CPU")
```

Comment:
I'm still getting this segfault with the fp32 C++ implementation.

Reply:
That's because the input shape is incorrect, I've change the input tensor to a more reasonable shape. It could pass now.

Reply:
I think the shape was OK but there's a bug that comes up with the specific data; seemed like a division by zero happens. Either way out of scope for this PR.

Reply:
I see, we can have a seperate PR to fix the op issue. But for this test, it enabled double compress, so I don't think quantize a (64,) size tensor is what we want, the current shape (300, 400) is more reasonable. Please let me know if you still want to keep the original (64,) shape.

Reply:
Let's keep the new shape and I'll note for myself that the old shape just happened to surface a data-dependent bug in the C++ CPU implementation.